data-model/publication-flow.md

```diff
@@ -103,7 +108,7 @@ Iedere aanvraag-activiteit gebruikt (`prov:used`) stukken. Deze worden als bijla
 **TODO**: regelgevend document als referentiedocument. Verder uit te werken wanneer designs gefinaliseerd zijn
 
 ## Niet via MR
-In het geval een publication-flow opgestart wordt niet via een MR wordt er een nieuw dossier (`dossier:Dossier`) gemaakt op het moment dat de publicatieaangelegenheid gemaakt wordt. De gebruiken kan vrij een titel (`dct:title`) en korte titel (`dct:alternative`) ingeven.
+In het geval een publication-flow opgestart wordt "niet via een MR" wordt er een nieuw dossier (`dossier:Dossier`) gemaakt op het moment dat de publicatieaangelegenheid gemaakt wordt. De gebruiker kan vrij een titel (`dct:title`) en korte titel (`dct:alternative`) ingeven. ((TBC hier willen we ook link aan het beleidsdomein en het type regelgeveing))
```

Comment:
In deze sectie wordt vooral het onderscheid gemaakt tussen publication-flow "niet via MR" en "wel via MR". Zijn beleidsdomein en type regelgeving iets dat enkel op publication-flow "niet via MR" ingevuld moeten worden? 
Of bedoel je dat ze bij "wel via MR" automatisch doorstromen vanuit de besluitvormingsaangelegheid die al gecapteerd wordt in Kaleidos?